ui/job-view/pushes/JobGroup.jsx

```diff
@@ -111,7 +111,7 @@ export default class JobGroup extends React.Component {
         } else {
           const lastJobSelected = {};
           countInfo = { ...countInfo, ...stateCounts[countInfo.btnClass] };
-          if (!_.isEmpty(lastJobSelected.job) && (lastJobSelected.job.id === job.id)) {
+          if (!isEmpty(lastJobSelected.job) && (lastJobSelected.job.id === job.id)) {
```

Comment:
Sheesh, totally true!  This bug has been there a while, but I've fixed it so that it will now show the "selection" on a count if the counts are collapsed, but the job within the count is selected.  Thanks for catching this!